infra/modules/event_bridge/main.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_cloudwatch_event_rule" "add_pet_lambda_event_rule" {
+  name                = "add-pet-lambda-rule"
+  schedule_expression = "cron(0 8 ? * * *)"
+}
+
+resource "aws_cloudwatch_event_target" "add_pet_lambda_target" {
+  arn  = var.lambda_function_arn
+  rule = aws_cloudwatch_event_rule.add_pet_lambda_event_rule.name
+}
+
+resource "aws_lambda_permission" "allow_cloudwatch_to_add_pet_lambda" {
```

Comment:
this will be a reusable module, not just for add_pet_lambda

use a naming convention that will be suitable for different lambdas